src/mjolnir/graphtilebuilder.cc

```diff
@@ -890,13 +890,14 @@ void GraphTileBuilder::AddTileCreationDate(const uint32_t tile_creation_date) {
 
 // return this tiles' edges' bins and its edges' tweeners' bins
 using tweeners_t = std::unordered_map<GraphId, std::array<std::vector<GraphId>, kBinCount>>;
-std::array<std::vector<GraphId>, kBinCount> GraphTileBuilder::BinEdges(const GraphTile* tile,
-                                                                       tweeners_t& tweeners) {
+std::array<std::vector<GraphId>, kBinCount>
+GraphTileBuilder::BinEdges(const boost::intrusive_ptr<const GraphTile>& tile, tweeners_t& tweeners) {
   std::array<std::vector<GraphId>, kBinCount> bins;
   // we store these at the highest level
   auto max_level = TileHierarchy::levels().back().level;
   // skip transit or other special levels and empty tiles
-  if (tile->header()->graphid().level() > max_level || tile->header()->directededgecount() == 0) {
+  if (!tile || tile->header()->graphid().level() > max_level ||
```

Comment:
we never call these functions with null tiles, if we do we dont want to hide a bug upstream. if you want to add stuff like this change it to an assertion that the tiles arent null

Reply:
Done.